driver.go

```diff
@@ -456,6 +463,13 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 		} else if strings.HasPrefix(driverConfig.NetworkMode, "container:") {
 			createOpts.ContainerNetworkConfig.NetNS.NSMode = api.FromContainer
 			createOpts.ContainerNetworkConfig.NetNS.Value = strings.TrimPrefix(driverConfig.NetworkMode, "container:")
+		} else if strings.HasPrefix(driverConfig.NetworkMode, "ns:") {
+			createOpts.ContainerNetworkConfig.NetNS.NSMode = api.Path
+			createOpts.ContainerNetworkConfig.NetNS.Value = strings.TrimPrefix(driverConfig.NetworkMode, "ns:")
+		} else if strings.HasPrefix(driverConfig.NetworkMode, "task:") {
+			createOpts.ContainerNetworkConfig.NetNS.NSMode = api.FromContainer
+			otherContainerName := fmt.Sprintf("%s-%s", strings.TrimPrefix(driverConfig.NetworkMode, "task:"), cfg.AllocID)
```

Comment:
Can we create a shared method to derive a container name from task name+allocID? I worry that should we ever need to change how the name is built it could be whack a mole trying to find all the Sprintf instances (a game I've had the displeasure to play).

Reply:
@nickethier : sure. I was lazy but you caught me ;-)

Reply:
I added a tool function, ok now?